docs/zh/latest/plugins/public-api.md

```diff
@@ -4,7 +4,7 @@ keywords:
   - APISIX
   - API 网关
   - Public API
-description: 本文介绍了 public-api 的相关操作，你可以使用 public-api 插件保护你需要暴露的 API 的端点。
+description: `public-api` 插件公开了一个内部 API 端点，使其可被公开访问。该插件的主要用途之一是公开由其他插件创建的内部端点。
```

Comment:
ditto, let's see if it resolves this error

<img width="1045" alt="image" src="https://github.com/user-attachments/assets/84dfdc22-d544-4fc2-bcc1-f470ca243993" />